### PR TITLE
Change some 'RC' keywords to more appropriate 'manual_control' if also valid for joystick

### PR DESCRIPTION
**Pull request by sfuhrer**
Most of all the arming/disarming reason should be "Stick gesture" and not "RC".

### CODE REVIEW

File: src/modules/rover_differential/module.yaml [hunk 1]
```diff
@@ -20,7 +20,7 @@ parameters:
         description:
           short: Manual yaw rate scale
           long: |
-            In manual mode the setpoint for the yaw rate received from the rc remote
+            In manual mode the setpoint for the yaw rate received from the yaw stick
```
Comment (sfuhrer) [inline src/modules/rover_differential/module.yaml:23]: fyi @chfriedrich98

Comment (chfriedrich98) [reply src/modules/rover_differential/module.yaml:23]: Makes sense to me :+1:

**Review by MaEtUgR — COMMENTED** [review]
Thanks for bringing this up. The wording is actually wrong.
The only thing I'd do differently is take a term e.g. "stick gesture" and then use that term everywhere and not have the one enum "manual_control" and the UI string "stick" and another enum "rc" for the same thing.

I can add a suggestion.

File: src/lib/events/enums.json [hunk 2]
```diff
@@ -448,7 +448,7 @@
                         },
                         "1": {
                             "name": "rc_stick",
```
[the remark this answers is not in the document]
Comment (MaEtUgR) [reply src/lib/events/enums.json]: 85394db7c029b96b09b713b2cce41a38da67eb9f

File: src/modules/commander/Commander.cpp [hunk 3]
```diff
@@ -503,7 +503,7 @@ static constexpr const char *arm_disarm_reason_str(arm_disarm_reason_t calling_r
 	switch (calling_reason) {
 	case arm_disarm_reason_t::transition_to_standby: return "";
 
-	case arm_disarm_reason_t::rc_stick: return "RC";
+	case arm_disarm_reason_t::rc_stick: return "Stick gesture";
 
 	case arm_disarm_reason_t::rc_switch: return "RC (switch)";
```
[the remark this answers is not in the document]
Comment (MaEtUgR) [reply src/modules/commander/Commander.cpp]: 85394db7c029b96b09b713b2cce41a38da67eb9f

File: src/modules/commander/Commander.cpp [hunk 4]
```diff
@@ -503,7 +503,7 @@ static constexpr const char *arm_disarm_reason_str(arm_disarm_reason_t calling_r
 	switch (calling_reason) {
 	case arm_disarm_reason_t::transition_to_standby: return "";
 
-	case arm_disarm_reason_t::rc_stick: return "RC";
+	case arm_disarm_reason_t::rc_stick: return "Stick gesture";
```
[the remark this answers is not in the document]
Comment (MaEtUgR) [reply src/modules/commander/Commander.cpp]: 85394db7c029b96b09b713b2cce41a38da67eb9f

File: src/modules/logger/params.c [hunk 5]
```diff
@@ -59,7 +59,7 @@ PARAM_DEFINE_INT32(SDLOG_UTC_OFFSET, 0);
  * @value 0 when armed until disarm (default)
  * @value 1 from boot until disarm
  * @value 2 from boot until shutdown
- * @value 3 depending on AUX1 RC channel
+ * @value 3 depending on manual control switch AUX1
```
[the remark this answers is not in the document]
Comment (MaEtUgR) [reply src/modules/logger/params.c]: 2dc567e4220796dc1ec3212e74c04cf18638251c

**Review by MaEtUgR — APPROVED** [review]
@sfuhrer I thought "joystick gesture" is even more clear. What do you think?

**Comment by sfuhrer** [comment]
> @sfuhrer I thought "joystick gesture" is even more clear. What do you think?

Not necessarily. I associate the word "joystick" to a gamepad generally, so a source for manual control inputs not being an RC. That's also how the groundstation tab is called:
image
So I would prefer to go back to "stick".

**Review by sfuhrer — COMMENTED** [review]
Thanks for the add-ons @MaEtUgR !

**Review by MaEtUgR — APPROVED** [review]
> associate the word "joystick" to a gamepad

You're right, that would be confusing again.

Just blew my mind how inaccurate humans including me usually are with wording 😄
How's a joystick a gamepad but not the stick of a remote? Does the prefix joy change it?
Because there were gameport "joystick"s for flight simulators in the 90s?
I digress 😇

**Pull request merged**